lib/alephant/broker/response/base.rb

```diff
@@ -61,6 +63,10 @@ def add_etag_allow_header
           })
         end
 
+        def self.options?(request_env)
+          request_env && request_env.respond_to?(:options?) && request_env.options?
```

Comment:
What does `&.` mean? I noticed request_env was sometimes `nil`, so I had to workaround it. After the elections, I'll be able to look at this further, as the logic is all quite tangled, and not straightforward to follow.